invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -82,7 +82,8 @@ export const RDMRecordResultsListItem = ({ result, index }) => {
           ))}
         </Item.Meta>
         <Item.Description>
-          {_truncate(description, { length: 350 })}
+          <div dangerouslySetInnerHTML={{__html: _truncate(description, { length: 350 })}}>
```

Comment:
We should strip HTML tags, and afterwards truncate instead of rendering the HTML - people will be able to screw up the frontpage/search quite a lot if we allow it.